stripe/src/main/java/com/stripe/android/view/PaymentMethodsActivityStarter.kt

```diff
@@ -37,6 +37,13 @@ class PaymentMethodsActivityStarter : ActivityStarter<PaymentMethodsActivity, Ar
         REQUEST_CODE
     )
 
+    constructor(activity: Activity, args: Args) : super (
```

Comment:
I tried just calling the full constructor (presumably defined on the superclass) but didn't compile. Is this the right way?

Reply:
You should pass the args into `startForResult()`, not as a constructor argument.